spec/sections/copies.bs

```diff
@@ -29,8 +29,16 @@ and "immediate" {{GPUQueue}} operations:
 - {{GPUQueue/copyExternalImageToTexture()}}, for copies from Web Platform image sources to textures
 
 Some texel values have <dfn dfn lt='equivalent texel representation'>multiple possible representations</dfn>
-of some values, e.g. as `r8snorm`, -1.0 can be represented as either -127 or -128.
-Copy commands are not guaranteed to preserve the source's bit-representation.
+of some values. As a result copy commands are not guaranteed to preserve the source's bit-representation.
+Examples include:
+
+ - snorm values may represent -1.0 as either -127 or -128.
+ - Any NaN or Infinity may be represented by an indeterminant, non-signaling NaN.
+ - Any subnormal value, -0.0, or +0.0 may be represented by either -0.0 or +0.0.
+
+Note: Copies may be performed with WGSL shaders, which means that any of the documented
+[WGSL IEEE-754 behaviors](https://gpuweb.github.io/gpuweb/wgsl/#differences-from-ieee754) may be
```

Comment:
s/`IEEE-754`/`floating point` because specifically we're referring to the _non_-IEEE-754 behaviors from WGSL.